pom.xml

```diff
@@ -294,7 +294,7 @@
         <hive.version>0.14.0</hive.version>
         <hadoop.version>2.6.1</hadoop.version>
         <hdfs.version>${hadoop.version}</hdfs.version>
-        <hbase.version>1.1.12</hbase.version>
+        <hbase.version>1.4.4</hbase.version>
```

Comment:
Since we are upgrading anyway, do we want to attempt Hbase 2.0 ?

Reply:
I am happy to give it a try.  I was a bit cautious with it being a major version change, and that there was no 2.0.1 yet.  But it does fit better with going to hadoop 3.1 coming in another pull request.  Probably after this one is merged in.

Reply:
In 2.0 TokenUtil is a part of hbase-server.  I will spend some time to see if there is something I can do to work around this but I really don't want to require the server code be shipped with all of the clients.  Perhaps I can refactor autocreds so the nimbus portions are split off into a separate package.